src/index.js

```diff
@@ -156,11 +156,19 @@ async function checkForNewSubmissions() {
       const cleanRecords = records
         .filter((r) => {
           if (typeof r.get("Name") === "undefined") return false;
-          if (r.get("Posted to Slack?") === "yes") return false;
+          // checking if posted to slack ANS record has a reminder set, if the date/time of reminder has
+          // already passed, and if the reminder has already been posted
+          if (
+            r.get("Posted to Slack?") === "yes" &&
```

Comment:
Let's have a separate checks for `r.get("Posted to Slack?") === "yes"` if no reminder set, and one for if the reminder is set (which is the check you've written).

Reply:
This check is only for filtering records we don't want to include in processing.
Currently this checks:
If posted to slack = yes
AND
(if no reminder is set OR reminder time has not yet passed OR reminder posted =yes )

I'm not sure I understand the need for having separate checks in this situation. Would you be able to elaborate on this?

Reply:
@azlyth Broke up the if statement so we're checking if the reminder has been set separately

Reply:
I guess what I'm really trying to do is separate reminder and new request processing, since they don't really need to be intertwined. So something like:
- start processing row
- if new request, handle it
- if not and this row needs reminder handling, do it

But this is clear enough, thanks for the update.